kansha/board/view.py

```diff
@@ -140,10 +139,10 @@ def render_Board_num_matches(self, h, comp, *args):
     if self.card_matches:
 
         if None in self.card_matches:
```

Comment:
`card_matches`  is a list which contains the ids of the cards that match the filter, or only None if no cards match the filter. If `card_matches` itself evaluates to False, that means the filter is off.